koku/koku/settings.py

```diff
@@ -52,8 +52,7 @@
 )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.getenv('DJANGO_DEBUG', True)
-DEBUG = (DEBUG == 'True' or DEBUG is True)
+DEBUG = bool(os.getenv('DJANGO_DEBUG', 'True'))
```

Comment:
This is actually complaining about the default value given to the getenv call. This will work
```
DEBUG = os.getenv('DJANGO_DEBUG')
DEBUG = (DEBUG == 'True' or DEBUG is True)
```